pallets/ajuna-awesome-avatars/src/migration/v6/mod.rs

```diff
@@ -521,10 +524,16 @@ pub mod mbm {
 						(old_config.stats.trade.bought, old_config.stats.trade.sold),
 					);
 
+					let default_locks = match season_id {
+						1 | 2 =>
+							Some(Locks { avatar_transfer: true, set_price: true, affiliate: false }),
+						_ => None,
+					};
```

Comment:
Ah do we have more seasons already, or is this just a safety check?

Reply:
We have 2 previous seasons this are the two in which the changes need to be made

Reply:
Alright, yeah, this is why I thought that we can just always set it to true, instead of checking the season it. But all good. 👍

Reply:
That looks very complex for something that only needs to be done during the migration.